src/main/java/edu/harvard/iq/dataverse/api/Dataverses.java

```diff
@@ -1969,4 +1969,15 @@ public Response createTemplate(@Context ContainerRequestContext crc, String body
             return e.getResponse();
```

Comment:
Can you please add a release note snippet? Please see https://guides.dataverse.org/en/6.8/developers/version-control.html#writing-a-release-note-snippet

Reply:
Thanks!